spec/models/miq_policy_content_spec.rb

```diff
@@ -0,0 +1,23 @@
+describe MiqPolicyContent do
+  describe ".seed" do
+    it "should contain conditions" do
+      [MiqAction, MiqEventDefinition, MiqPolicy, MiqPolicyContent].each(&:seed)
+
+      specifications = YAML.load_file(File.join(ApplicationRecord::FIXTURE_DIR, "miq_policy_contents.yml"))
+      specifications_by_pkey = specifications.index_by do |h|
+        "#{h[:miq_policy]}_#{h[:success_sequence]}_#{h[:failure_sequence]}"
+      end
+
+      count = 0
+      MiqPolicyContent.all.each do |mpc|
+        spec = specifications_by_pkey["#{mpc.miq_policy.name}_#{mpc.success_sequence}_#{mpc.failure_sequence}"]
+        expect(mpc).to have_attributes(
+          spec.except(:created_on, :updated_on, :miq_policy, :miq_action, :miq_event_definition)
```

Comment:
In other words, I don't know what we gain from testing all of these attributes over just checking `:name` or whatever if useful.  We don't really have to test attributes= and save except for things we're concerned about (missing attributes aren't updated, unchanged attributes aren't saved, etc.)